lxd/patches.go

```diff
@@ -188,6 +190,29 @@ func patchesApply(d *Daemon, stage patchStage) error {
 
 // Patches begin here
 
+func patchDnsmasqEntriesIncludeDeviceName(name string, d *Daemon) error {
+	return d.cluster.InstanceList(nil, func(inst db.Instance, proj db.Project, profiles []api.Profile) error {
```

Comment:
We should use whatever the equivalent function is to only query the local instances.
This one will list everything in an entire cluster rather than just for the local system.

Reply:
There doesn't seem to be a specific `db` function for listing instances on the current node. Instead I've used `d.cluster.GetNodeID` to get the id of the node and passed that into the `InstanceFilter` (first argument of the `InstanceList` function).